draft-ietf-rats-architecture.md

```diff
@@ -373,60 +378,78 @@ Attesting Environments are designed specifically with claims collection in mind.
-
-{{layered}} depicts an example of a device that includes (A) a BIOS stored
-in read-only memory in this example, (B) an updatable bootloader, and (C)
-an operating system kernel.
+Attester -- in itself -- is composed and therefore has to conduct the Claims collection
+in order to create believable Attestation Evidence.
+The following example is intended to illustrate this composition:
 
+A very common example is elaborated on to illustrate Layered Attestation.
 {:layered: artwork-align="center"}
 ~~~~ LAYERED
 {::include layered-attester.txt}
 ~~~~
 {:layered #layered title="Layered Attester"}
 
-Attesting Environment A, the read-only BIOS in this example,
-has to ensure the integrity of the bootloader (Target Environment B).
-There are
-potentially multiple kernels to boot, and the decision is up to the bootloader.
-Only a bootloader with intact integrity will make an appropriate decision. Therefore, these Claims have to be measured securely.
-At this stage of the boot-cycle of the
-device, the Claims collected typically cannot be composed into Evidence.
-
-After the boot sequence is started, the BIOS conducts the
-most important and defining feature of layered attestation, which is that
-the successfully measured Target Environment B
-now becomes (or contains) an Attesting Environment for the next layer.
-This procedure in Layered Attestation is sometimes called "staging".
-It is important that the new Attesting Environment B not be
-able to alter any Claims about its own Target Environment B.
-This can be ensured having those Claims be either signed by Attesting
-Environment A or stored in an untamperable manner by Attesting
-Environment A.
-
-Continuing with this example, the bootloader's Attesting Environment B is now in charge of collecting Claims
-about Target Environment C, which in this example
-is the kernel to be booted.  The final Evidence thus contains two sets of
-Claims: one set about the bootloader as measured and signed by the BIOS,
-plus a set of Claims about the kernel as measured and signed by the bootloader.
-
-This example could be extended further
-by, say, making the kernel become another Attesting Environment for
-an application as another Target Environment, resulting in a third set
-of Claims in the Evidence pertaining to that application.
-
-The essence of this example is a cascade of staged environments. Each
-environment has the responsibility
-of measuring the next environment before the next environment is started.
-In general, the number of layers may vary by device or implementation,
-and an Attesting Environment might even have multiple Target Environments
-that it measures, rather than only one as shown in {{layered}}.
+The very first Attesting Environment has to ensure the integrity of
+the (U)EFI / BIOS / Firmware that initially boots up a composite device (e.g.,
+a cell phone).
+
+~~~
+    Henk: we are looking for a better term than UEFI/BIOS/Firmware
+~~~
```

Comment:
This seems to revert a bunch of changes we agreed to.   Maybe it needs a rebase or something?

Reply:
Yes.  All of these changes have now been reverted.